src/pyhf/cli/rootio.py

```diff
@@ -8,10 +9,18 @@
 logging.basicConfig()
 log = logging.getLogger(__name__)
 
+# This is only needed for Python 2/3 compatibility
+def _ensure_dirs(path):
+    try:
+        os.makedirs(path, exist_ok=True)  # lgtm [py/call/wrong-named-argument]
+    except TypeError:
+        if not os.path.exists(path):
+            os.makedirs(path)
+
```

Comment:
@lukasheinrich This shouldn't have been added as this was removed during the Python 3 only migration.